docket/common/utils.py

```diff
@@ -1,5 +1,252 @@
+from datetime import datetime, timedelta
+from fcntl import flock, LOCK_EX, LOCK_NB
+from collections import OrderedDict, Mapping, namedtuple
+from socket import inet_pton, AF_INET6, AF_INET, error as SocketError
+from flask import request, has_request_context
+import os
+import yaml
+import re
+
+# import docket configuration
+from config import Config
+
+FreeSpace = namedtuple('FreeSpace', 'bytes nodes' )
```

Comment:
let's make this a sequence of strings, e.g. `['bytes', 'nodes']`. I think it makes a bit more clear what's going on here.

Also, this is really cool. I haven't used namedtuples before.